README.md

```diff
@@ -161,10 +175,25 @@ pip install .
 ## 2. Sample Sequences Using Monte Carlo
 
 **File**: `sample.py`
-**Description**:  
 
+**Description**:
+  
 Uses Monte Carlo sampling to sample sequences from a  probability distribution. A temperature factor can be applied to affect the distributions. It will return a `.fasta` file and/or a `.json` file with the sequences and a `.csv` file with basic sequence metrics such as isoelectric point, molecular weight and charge. Further metrics can be calculated using NetSolP-1.0 (see `scripts/run_netsolp.sh`).
 
+### 2.1. Generating the input files for Monde Carlo sampling
+
+Run the prediction step using predict.py as instructed in section 1.2.
+
+
+### 2.2 Run:
+
+```
+python sample.py --path_to_pred_matrix /PATH/TO/{MODEL_NAME}.csv --path_to_datasetmap /PATH/TO/{MODEL_NAME}.txt --seed 42 --workers 35 --sample_n 200 --temperature 0.1
+```
+Replace the path placeholders with the actual paths to your files generated in the prediction step.
+
+This will sample 200 sequences using the provided probability distribution and temperature factor of 0.1. A higher temperature will lead to more diverse sequences being sampled with the temperature range being 0.1-5. Output sampled sequences can be increased or decreased by adjusting the number of output structures ''--sample_n 200''.
+
```

Comment:
I think you could write a little bit about sampling temperature. Maybe explaining each of the flags makes it easier to follow. 

Temperature can range from 0 to any number. I would write something like (ofc needs to be written better)

- 0 is equivalent to argmax
- Exploitation: A low temperature value e.g. 0.2 will move the distribution towards the argmax so the sequences generated will be very similar and close to the argmax. 
- Exploration: A higher temperature e.g. 1 will keep the distribution as is, meaning that the sequences generated will generally have high diversity. 

(I'm sure there's a nicer way to phrase these points)